src/components/UserDetailSections/ReadingRoomAccess/ReadingRoomAccess.js

```diff
@@ -22,19 +24,60 @@ const ReadingRoomAccess = (props) => {
     accordionId,
     expanded,
     onToggle,
-    userRRAPermissions,
+    readingRoomPermissions,
   } = props;
   const intl = useIntl();
-  const [filteredRRA, setFilteredRRA] = useState([]);
+  const userRRAPermissions = useMemo(() => readingRoomPermissions?.records, [readingRoomPermissions]);
+  const { isPending } = readingRoomPermissions;
+  const visibleColumns = [
+    rraColumns.ACCESS,
+    rraColumns.READING_ROOM_NAME,
+    rraColumns.NOTES,
+    rraColumns.ID
+  ];
+  const columnMapping = {
+    [rraColumns.ACCESS]: <FormattedMessage id="ui-users.readingRoom.access" />,
+    [rraColumns.READING_ROOM_NAME]: <FormattedMessage id="ui-users.readingRoom.name" />,
+    [rraColumns.NOTES]: <FormattedMessage id="ui-users.readingRoom.note" />,
+    [rraColumns.ID]: <FormattedMessage id="ui-users.readingRoom.lastUpdated" />,
+  };
```

Comment:
If you anticipate these lists remaining linked, you might be better served by 
```
const columnMapping = {...};
const visibleColumns = Object.keys(columnMapping);
```
[The traversal order of keys in an object is stable and well-defined](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in#description) and matches what you want here: order of property creation.